extensions/typescript/src/features/jsDocCompletionProvider.ts

```diff
@@ -32,7 +32,7 @@ class JsDocCompletionItem extends CompletionItem {
 	) {
 		super('/** */', CompletionItemKind.Snippet);
 		this.detail = localize('typescript.jsDocCompletionItem.documentation', 'JSDoc comment');
-		this.insertText = '';
+		this.insertText = ' ';
```

Comment:
This is the minimal change that fixes the issue. I also considered updating the suggestion provider to not apply the new enter logic for empty insert text or for completions with commands, however this carried more risk.

I'm also going to try to get TypeScript to improve the api we use to get doctemplates so that the `JsDocCompletionItem` doesn't require using a command